src/Routing/Duplex.purs

```diff
@@ -128,6 +167,16 @@ as f g (RouteDuplex enc dec) = RouteDuplex (enc <<< f) (Parser.as identity g dec
 int :: RouteDuplex' String -> RouteDuplex' Int
 int = as show Parser.int
 
+-- | Make Boolean codec out of a String codec.
```

Comment:
It's hard to come up with a good descriptions of what those functions of type `RouteDuplex' a -> RouteDuplex' b` do without talking about printing and parsing separately :P

Reply:
Maybe "restricts" or "refines"?

> Refines a codec of Strings to Booleans, where `true` and `false` are the strings "true" and "false".

What do you think?